llvm/lib/Target/SPIRV/SPIRVUtils.cpp

```diff
@@ -205,6 +205,10 @@ addressSpaceToStorageClass(unsigned AddrSpace, const SPIRVSubtarget &STI) {
                : SPIRV::StorageClass::CrossWorkgroup;
   case 7:
     return SPIRV::StorageClass::Input;
+  case 8:
+    return SPIRV::StorageClass::Output;
+  case 9:
+    return SPIRV::StorageClass::Private;
```

Comment:
There are some potential synchronization issues between the repositories that we should address in the future. Would you mind to move Private to address space 10? The reason I'm asking for this is that the [llvm-spirv translator uses it for CodeSectionINTEL storage class](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/88b18319c943e70c2475b495806226e117fdaa15/lib/SPIRV/SPIRVInternal.h#L192) :)

Reply:
Sure